package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I'm not familiar with what this is so a brief comment at the top would be helpful

Reply:
It's a config file from NPM.
Mainly for "npm install" to install all the dependencies.